code/modules/shuttle/mini_dropship.dm

```diff
@@ -152,12 +152,12 @@
 	if(damaged)
 		return
 	X.visible_message("[X] begins to slash delicately at the computer",
-	"You start slashing delicately at the computer.")
+	"We start slashing delicately at the computer. This will take a while.")
```

Comment:
just for next time, we are not much going to keep the "we" stuff and all, the stuff can be targeted at the player just fine, we decided that some time ago

Reply:
okay